test/tests/api/v2_0/workflows_tests.py

```diff
@@ -0,0 +1,213 @@
+
+        # Getting the identifier of the first workflow in order to validate the get-id function
+        Api().workflows_get()
+        rawj = json.loads(self.__client.last_response.data)
+        instance_id = rawj[0].get('id')
+        assert_is_not_none(instance_id)
+        Api().workflows_get_by_id(instance_id)
+        assert_equal(200,self.__client.last_response.status)
+
+    @test(groups=['workflows_get_id_api2'],depends_on_groups=['workflows_get_api2'])
+    def test_negative_workflows_id_get(self):
+        """ Negative Testing GET:/workflows/identifier"""
+        try:
+            Api().workflows_get_by_id("WrongIdentifier")
+        except Exception,e:
+            assert_equal(404,e.status, message = 'status should be 404')
+
+    @test(groups=['test_workflows_graphs_get_api2'])
+    def test_workflows_graphs_get(self):
+        """Testing GET:/workflows/graphs"""
+        Api().workflows_get_graphs()
+        assert_equal(200,self.__client.last_response.status)
+        assert_not_equal(0, len(json.loads(self.__client.last_response.data)),
+                         message='Workflows list was empty!')
+
+    @test(groups=['workflows_graphs_put_api2'])
+    def test_workflows_graphs_put(self):
+        """ Testing PUT:/workflows/graphs """
+
+        # Make sure there is no workflowTask with the same name
+        Api().workflows_get_graphs_by_name('*')
+        rawj = json.loads(self.__client.last_response.data)
+
+        for i, var in enumerate(rawj):
+            if self.workflowDict['injectableName'] == str(rawj[i].get('injectableName')):
+                fnameList = str(rawj[i].get('friendlyName')).split('_')
+                suffix = int(fnameList[1]) + 1
+                self.workflowDict['friendlyName'] = fnameList[0] + '_' + str(suffix)
+                break
+
+        # Add a workflow task
+        LOG.info ("Adding workflow task : " + str(self.workflowDict))
+        Api().workflows_put_graphs(body=self.workflowDict)
+        resp = self.__client.last_response
+        assert_equal(201,resp.status)
+
+        # Validate the content
+        Api().workflows_get_graphs()
+        rawj = json.loads(self.__client.last_response.data)
+        foundInsertedWorkflow = False
+        for i, var in enumerate(rawj):
+            if self.workflowDict['injectableName'] == str(rawj[i].get('injectableName')):
+                foundInsertedWorkflow = True
+                readWorkflowTask = rawj[i]
+                readFriendlyName = readWorkflowTask.get('friendlyName')
+                readInjectableName = readWorkflowTask.get('injectableName')
+                assert_equal(readFriendlyName,self.workflowDict.get('friendlyName'))
+                assert_equal(readInjectableName,self.workflowDict.get('injectableName'))
+
+        assert_equal(foundInsertedWorkflow, True)
+
+    @test(groups=['workflows_graphs_get_name_api2'],
+          depends_on_groups=['workflows_graphs_put_api2'])
+    def test_workflows_library_id_get(self):
```

Comment:
Should probably change the test function to something like `test_workflows_graphs_get_by_name`